2-mandatory/8-password-validator.js

```diff
@@ -22,8 +22,20 @@ Expected Result:
 PasswordValidationResult=  [false, false, false, false, true]
 
 */
+validatePasswords(["StUFf27%", "Pl3nty!", "Jai33", "shajsaUA**&&", "Pl3nty!"]);
+const validatePasswords = (passwordArray) => passwordArray.map(isValidPassword);
 
-function validatePasswords(passwords) {}
+const isValidPassword = (password, index, array) =>
+  password.length >= 5 &&
+  containsUppercaseLetter(password) &&
+  containsLowercaseLetter(password) &&
+  containsNumber(password) &&
+  containsSymbol(password) &&
+  array.findIndex((element) => element === password) === index
+    ? true
```

Comment:
as I know no need to give true or false here (? true: false) function automatically returns you true or false if conditions up will match the given argument!

Reply:
This is true! I admit that I used to do this all time but realised it was not necessary only a few months ago 😅